### PR TITLE
ADDDED NEW QUESTION

### PR DESCRIPTION
**Pull request by scamander27**

### CODE REVIEW

**Comment by sanjeevyadavIT** [comment]
hi @scamander27  Thank you so much for the your contribution 🎉

**Comment by rishabh115** [comment]
hi @scamander27 ,  this is quite an unhandy question to ask before accepting the job offer.

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ At the time of joining, the HR potrays the company like it is match made in Heav
 | 10 | Usually help half or whole financially for IT events and certifications?|
 | 11 | Can you describe the culture of the company?|
 | 12 | How easily the upper management is approachable for suggestion or queries?|
+| 19 | Can I work from home on Fridays?|
```
Comment (sanjeevyadavIT) [inline README.md:22]: This question, might not be good for first impression towards the interviewer, could you please change the question.

**Review by sanjeevyadavIT — CHANGES_REQUESTED** [review]
Hi @scamander27, We have requested some changes, please go through the review, and makes changes accordingly :hammer: 

We would be happy to merge your PR after the changes :rocket:

**Comment by sanjeevyadavIT** [comment]
Hi @scamander27 there are some changes requested in the PR, if no activity is found within 7 days, this PR will be closed.

**Pull request closed without merging**